src/Playwright.Tests/.runsettings

```diff
@@ -0,0 +1,5 @@
+<RunSettings>
+   <NUnit>
+       <NumberOfTestWorkers>24</NumberOfTestWorkers>
```

Comment:
I have 5! Let's leave defaults that count number of cores.

Reply:
This one shouldn't have been commited. Removing...